python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1466,8 +1508,11 @@ def _transform_mask(stride_dim, ellipsis_mask):
         fshape_indices = None
         if begin_mask or end_mask or ellipsis_mask or new_axis_mask or shrink_axis_mask:
             begin, end, stride, fshape_indices = _transform_mask(stride_dim, ellipsis_mask)
-        out = _op.strided_slice(inputs[0], begin=begin, end=end, strides=stride)
-        out_shape = _infer_shape(out, mod)
+        out = _op.strided_slice(inputs[0],
+                                begin=_expr.const(begin),
+                                end=_expr.const(end),
+                                strides=_expr.const(stride))
```

Comment:
Don't need _expr.const for begin, end and strides, since we allow normal python list to be passed.